deploy/lib/setIamPolicy.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const _ = require('lodash');
+const BbPromise = require('bluebird');
```

Comment:
Let's stick just to native promises (note we can fully use async/await). I believe we should get rid of `bluebird`, as we plant do so in a Framework).

If you find it difficult with native promises, let me know, I'll try to help

Reply:
I've tried to move to using native promises, but since Im not really experienced in the area and don't have other examples to go by, I haven't been able to get it to work quite right. If you have an example somewhere of what changing from bluebird to native looks like, it'd be much appreciated!